src/components/ExpandableSection/ExpandableSection.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import { Button, ButtonIcon } from '../../';
+
+class ExpandableSection extends Component {
+  constructor(props) {
+    super(props);
+
+    const { open, collapsable } = props;
+
+    if (!open && !collapsable) {
+      // eslint-disable-next-line
+      console.warn(
+        '[react-lds] ExpandableSection:',
+        'When collapsable is false, open cannot be false.',
+      );
+    }
+
+    this.state = {
```

Comment:
This component should be usable as a `Controlled` and an `Uncontrolled` component. If a component is uncontrolled, it manages its own state (what you are doing right now). The problem here is that there is no interface to open and close the section programatically. The way to fix this is as follow:

 - If `defaultOpen` is passed, apply it to `this.state` and and let the component handle it from there
 - If `open` is passed, use the `open` prop instead of the internal state and simply render it. 
 - If there is a conflict with collapsible, overwrite `defaultOpen` in uncontrolled mode and add a warning in controlled mode